Packages/make_general_adjustments

```diff
@@ -6,17 +6,24 @@ for f in `ls ${VERSIONING}/*`
   do
     source $f
   done
+if [ -f "customize.sh" ] ; then
+  source customize.sh
+fi
 cd Packages
 
 echo "** Performing general actions **"
 set -x
 sed -i "s/cape_universal=enable/consoleblank=0 fbcon=rotate:1 omap_wdt.nowayout=0/" /boot/uEnv.txt
-sed -i "s/arm/kamikaze/" /etc/hostname
-sed -i "s/arm/kamikaze/g" /etc/hosts
 sed -i "s/AcceptEnv LANG LC_*/#AcceptEnv LANG LC_*/"  /etc/ssh/sshd_config
-echo "** Set Root password to $ROOTPASS **"
-echo "root:$ROOTPASS" | chpasswd
-chown -R octo:octo `pwd`
+
+# adjust the hostname
+echo ${MASTER_HOSTNAME} >/etc/hostname
+sed -i '/arm/d' /etc/hosts
```

Comment:
Unless I miss my guess, this sed command won't run properly, as it's missing the operation header...
Shouldn't it be `sed -i 's/arm/d/'` instead?

Reply:
No. your suggestion replaces "arm" with "d". My code deletes the line containing "arm".

Reply:
ah - ok, I need to brush up on my sed syntax. I was under the impression the command was at the beginning of the string...

Reply:
Re: sourcing "customize.sh" multiple times. -- Because of the way some of the functions run, I think that we are required to source the versioning information in each of the "install" scripts. As a result, we need to allow the customization to be effective for each usage.

Re: install_flasher.  -- This may be an artifact of splitting a group of changes into multiple PRs because of the reluctance in accepting multiple effects in the same PR and having no control over the order in which they get merged. Since multiple PRs need to affect the same piece of code, an order needs to be established and the prerequisite ones processed rapidly.